src/common/tusb_verify.h

```diff
@@ -120,7 +120,9 @@
 #define TU_VERIFY_1ARGS(_cond)                         TU_VERIFY_DEFINE(_cond, , false)
 #define TU_VERIFY_2ARGS(_cond, _ret)                   TU_VERIFY_DEFINE(_cond, , _ret)
 
+#ifndef TU_VERIFY
```

Comment:
verify doesn't trigger any error, it is considered normal scenario for an VERIFY() to failed. Therefore I think we don't have a need to overwrite it.

Reply:
Agreed.